applications/CompressiblePotentialFlowApplication/python_scripts/compute_moment_process.py

```diff
@@ -0,0 +1,57 @@
+import KratosMultiphysics
+import numpy as np
```

Comment:
Use Kratos array types. `numpy` doesn't come as a default package of Python so it is possible that some users lack it.